objects.js

```diff
@@ -9524,4 +9525,132 @@ StagePrompterMorph.prototype.mouseClickLeft = function () {
 
 StagePrompterMorph.prototype.accept = function () {
     this.isDone = true;
+}
+
+// CamSnapshotDialogMorph /////////////////////////////////////////////////////////
```

Comment:
@bromagosa any particular reason why the CamSnapshotDialogMorph should be in ```objects.js``` rather than in ```gui.js```? Not that it means much. If it's okay with you I'd probably move it over myself... (?) Tnx!

Reply:
Hmm, no reason, really. I guess because it was about costumes, but it makes sense to have it in gui.js :)